java-frontend/src/main/java/org/sonar/java/se/checks/LocksNotUnlockedCheck.java

```diff
@@ -42,8 +43,16 @@
 @Rule(key = "S2222")
 public class LocksNotUnlockedCheck extends SECheck {
 
-  private enum LockStatus implements ObjectConstraint.Status {
+  public enum LockConstraint implements Constraint {
     LOCKED, UNLOCKED;
+
+    @Override
+    public String valueAsString() {
+      if(this == LOCKED) {
+        return "locked";
+      }
+      return "Unlocked";
```

Comment:
In fact what's amusing is that this value is never used as it is irrelevant when reporting...